recipe/build_base.sh

```diff
@@ -4,13 +4,6 @@ set -ex
 # Get an updated config.sub and config.guess
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* .
 
-if [[ ! -d ${BUILD_PREFIX}/python-bin ]]; then
```

Comment:
Why are you removing this? This actually adds some reproducibility to the build as otherwise it will depend on a Python "from the outside".